### ISSUE TITLE
Request Mine Factory Reloaded support

### ISSUE THREAD

**Issue by Sinhika** [GrowthcraftCE/Growthcraft-1.7#235]
I'd like the MFR Fruit Picker to work on Growthcraft Apple trees, GC Grapes, and GC Hops vines, and anything else appropriate.  I'm not sure how much work it would be to get the MFR Harvester and Planter to work with GC Rice in rice paddies, but it would be nice to have that, too.

**Comment by Ar97x** [comment]
Something I look forward to see. It would be a good start to automate booze
production.

**Comment by IceDragon200** [comment]
@Sinhika @Ar97x I've added support for Harvesters and Fruit Pickers, planters are a whole new ballgame for Rice, you can check the latest build

**Comment by IceDragon200** [comment]
:( Why is that no one reports back when I do these things

PING @Ar97x @Sinhika Is the update okay, can I close this issue?

**Comment by Ar97x** [comment]
I'm on tests week on college, so I don't have much time. Close it, if we
find bugs we can reopen it.

**Comment by Sinhika** [comment]
sorry, been busy. I didn't realize you were waiting on me to test it.

**Comment by Sinhika** [comment]
It crashed hard. Note that I just dropped the 3/17 build in as a replacement for the current release version of GrowthCraft, so it would have tried to read the existing config files. You may have a problem with gracefully transitioning from current configs to whatever new versions you are using.

Attached the crashlog.
[crash-2016-03-17_14.00.24-client.txt](https://github.com/GrowthcraftCE/Growthcraft-1.7/files/178501/crash-2016-03-17_14.00.24-client.txt)

**Comment by IceDragon200** [comment]
@Sinhika Just trash your `growthcraft/cellar/*.json` configs, or update them, there is no way for me to transition the JSON configs sensibly in java

**Comment by Sinhika** [comment]
Works now.

The Fruit picker works with apple trees, and correctly grab only ripe apples. The Harvester will shear leaves off apple tree (if told to) and harvest wood.  The Fruit picker works with Hops and Grapes, also correctly grabbing ripe hops & grapes.  The Planter will plant apple saplings (no surprise; it did before).

The Harvester will harvest rice. However, the Ferilizer does not recognize rice as a fertilizable crop. I got the impression that you meant that Planter support for rice was not implemented.  However, the Harvester leaves empty rice pads, so rigging up a dispenser with a water bucket allows them to be easily reflooded. Can Planter support be implemented to plant rice on an already flooded rice pad?

**Comment by IceDragon200** [comment]
@Sinhika I'll look into the Fertilizer and Planter for rice then

**Comment by IceDragon200** [comment]
@Sinhika Rejoice, MFR Planter now plants our rice, and the MFR Fertilizer can fertilize it.

We are still trying to figure out how to automate the paddy filling, without resorting to tiles

**Comment by Sinhika** [comment]
Okay, I tested it. Fertilzer works on GC rice, Planter will plant GC Rice in a water-filled paddy, and ignore dry paddies. Works as expected.

**Comment by IceDragon200** [comment]
@Sinhika We can close this right?

**Comment by Sinhika** [comment]
Okay by me.
